src/bsk_rl/scene/targets.py

```diff
@@ -48,10 +48,6 @@ def id(self) -> str:
             self._id = f"{self.name}_{id(self)}"
             return self._id
 
-    def __hash__(self) -> int:
```

Comment:
To check if I understood it right, the hash will now be generated based on the entire object, and not only on its id, correct?

Reply:
Yeah, I'm not sure why I put this in in the first place.

Reply:
Actually, it may be worth checking how targets are copied, if ever. I believe deepcopy changes the hash but not the id, so this could potentially lead to issues. I'll take a closer look at some point; not a big enough performance improvement to warrant potentially introducing an error without being sure of this.

Reply:
I hadn't considered that before, but I agree. This is a subtle change that can lead to unexpected consequences.